packages/components/src/modifiers/hds-code-editor.ts

```diff
@@ -412,6 +505,30 @@ export default class HdsCodeEditorModifier extends Modifier<HdsCodeEditorSignatu
     }
   );
 
+  private _setupEditorMutationObserver() {
+    this.mutationObserver = new MutationObserver((mutations) => {
+      mutations.forEach((mutation) => {
+        mutation.removedNodes.forEach((node) => {
+          if (!(node instanceof HTMLElement)) {
+            return;
+          }
+
+          const removedNodeContainsLintPanel =
+            node.querySelector('.cm-panel-lint') !== null;
```

Comment:
```suggestion
            node.querySelector('.cm-panels) !== null;
```
[Suggestion] Maybe this needs to be the parent class, since that's the outermost element getting remove? 🤷 
<img width="439" alt="Screenshot 2025-03-20 at 5 30 11 PM" src="https://github.com/user-attachments/assets/d384220c-9113-4ede-8372-531521f27caf" />

Reply:
Just missed a function call 🤦